src/document/DocumentAnnotator.ts

```diff
@@ -97,10 +114,21 @@ export default class DocumentAnnotator extends BaseAnnotator {
             return;
         }
 
-        if (mode === Mode.HIGHLIGHT) {
-            this.annotatedEl.classList.add('ba-is-highlighting');
-        } else {
-            this.annotatedEl.classList.remove('ba-is-highlighting');
+        ['ba-is-highlighting', 'ba-is-create-region'].forEach(
```

Comment:
We seem to have a lot of duplicates of these strings now. Should there be an annotation-type -> classname map?

Reply:
I think that would help. Where would this map live?